docs/content/en/docs/getting-started/production-environment/_index.md

```diff
@@ -213,6 +217,23 @@ Follow these steps if you want to use your initial cluster to create and manage
 
    To add more workload clusters, go through the same steps for creating the initial workload, copying the config file to a new name (such as `eksa-w02-cluster.yaml`), modifying resource names, and running the create cluster command again.
 
+1. ###### Install curated packages:
+
+   [cert-manager](https://cert-manager.io/) installation is required before installing curated packages on workload clusters. You can install cert-manager using the command below, then complete the curated packages installation by following instructions [here]({{< relref "../../tasks/packages" >}}).
+
+   ```bash
+   helm repo add jetstack https://charts.jetstack.io
```

Comment:
We are installing this from jetstack? What about our build? Also seems like this should be a new page linked out of the prod env page like that page you changed below